<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QSpinBox" name="spinBox_Encuestas_satisfaccion">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>270</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>210</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Encuesta</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Articulo_id">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>270</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Grado de satisfaccion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>140</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Costo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>200</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Agregar">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ventas_venta">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Encuestas_Fecha">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>240</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Articulo_Nombre">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>160</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Art√≠culo</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>240</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>100</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Mostrar">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Clientes</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Clientes</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ventas_Fecha">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>30</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ventas</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Cliente_nombre">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Costo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>70</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Articulo_Venta">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>320</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>180</y>
          <width>41</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Venta:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ventas_id">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Cliente_id">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Articulo_Costo">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>320</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Venta:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ventas_costo">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>140</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="salida">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>10</y>
          <width>281</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Mostrar_Articulos">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Articulos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Mostrar_Ventas">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ventas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Mostarr_Encuestas">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Encuestas</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Mostrar</string>
       </attribute>
       <widget class="QTableWidget" name="tabla">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>21</y>
          <width>681</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Clientes">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Clientes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Articulos">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Articulos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Ventas">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ventas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Encuestas">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>310</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Encuestas</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="actionCosto"/>
    <addaction name="actionVenta"/>
    <addaction name="actionSatisfaccion"/>
    <addaction name="actionCliente"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuSort"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionCosto">
   <property name="text">
    <string>Costo</string>
   </property>
  </action>
  <action name="actionVenta">
   <property name="text">
    <string>Venta </string>
   </property>
  </action>
  <action name="actionSatisfaccion">
   <property name="text">
    <string>Satisfaccion</string>
   </property>
  </action>
  <action name="actionCliente">
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
